AudiusProject/audius-protocol
# [ASI-962] Convert Solana track listen from hash to sorted set

### Description
* Uses a time series (redis sorted set) for the Solana track listen health check (Identity Service's `/tracks/listen/solana/status`)
* Keeps the old redis hash hourly bucketing for informational purposes
* Prunes expired sorted set entries when `/tracks/listen/solana/status` is hit (redis doesn't support expiring members in a sorted set, so this pruning makes more sense than pruning when `/tracks/:id/listen` is hit, since the latter is hit more often)
* Renames/adds the following fields returned from `/tracks/listen/solana/status` to use sorted set:
  * `success` -> `totalSuccessCount`
  * `submission` -> `totalSubmissionCount`
  * `percentSuccess `-> `totalPercentSuccess`
  * `recentSubmissionCount`
  * `recentSuccessCount`
  * `recentSuccessPercent`

### Tests
* See integration tests
* To test manually:
  * Pull the branch on your remote instance and point your client to it
  * Upload a track and listen to it
  * Go to the Identity Service's port 7000 on your remote instance and verify that you see correct values for the variables mentioned above. Here's mine: http://34.133.153.124:7000/tracks/listen/solana/status

### How will this change be monitored? Are there sufficient logs?
Pingdom will definitely let us know if there's an issue.
